### ISSUE TITLE
Extentions

### ISSUE THREAD

**Issue by katsikora** [maxplanck-ie/snakepipes#598]
- [x] snakemake 5.13 +(  --lint is currenlty disabled)
- [x] EmptyDrops scRNAseq
- [x] alevin (Ward)
- [x] rMATS turbo (Ward)
- [x] review settings for RNA velocity - we're doing good by providing velocyto counts as in the published paper. Providing alevin_sep_gtr or alevin_coll_gtr in the alevin mode seems like a good alternative. see Soneson et al. 2020 bioRxiv
- [x] scRNAseq genotyping + scSplit ? (not in this release)
- [ ] clean up the filter_bam + samtools +umi_tools jungle
- [ ] numeric sample names: handle conversion *inside* R scripts 
- [ ] update our internal organism.yaml with newer ensembl versions
- [ ] update the Rscript example command on top of  DESeq2.R etc. or add to docs
- [x] make --fromBAM work in the same way between workflows (necessary ??) (not in this release)
- [ ] add test data for preprocessing workflow and for umi-tools
- [ ] CSAW external bed as input
- [x] STARsolo synthesize report
- [ ] annotate emptyDrops object with STARsolo filter
- [ ] add bam filtering to ChIPseq
- [ ] docs
- [ ] News.rst
-------------------------------------------------------------------------------------------------------------------------------
- [x] add yaml check: check that the paths exist ( path to temp ) --->>> added to snakePipes info. 
- [x] do we need all these compound shell commands that result in the error not being reported in cluster logs?
- [x] replace as many bash file ops with python as possible; remove any linux-specific flags e.g. ln -r
- [x] put PCA plots back in the WGBS reports where missing
- [x] docs on running snakefiles without workflow wrappers
- [x] tecounts - if sampleSheet
- [x] noncoding-RNA-seq --fromBAM
- [x] multiQC
- [x] noncoding-RNA-seq DESeq2 report
- [x] add interval list and region analysis back to WGBS (only metilene)
- [x] add support for SE data to Genrich in ChIPseq
- [x] Genrich -S sort bam by queryname??? (not now - only if memory comes short)
- [x] add --skipQC to --fromBAM to skip redundant steps
- [x] meaninigful names for conda envs! ---->>> not compatible with snakemake, so not added
- [x] add user input options to DMR caller
- [x] fix TMPDIR if necessary
- [x] fix --verbose and logging in WGBS
- [x] check sampleSheet usage with CSAW
- [x] rename sampleSheet in scRNAseq mode Gruen
- [x] fix histone HMM output number formatting
- [x] News.rst
- [x] docs

**Comment by katsikora** [comment]
If anyone feels they want to take care of any of these points, go ahead and assign yourself.

**Comment by LeilyR** [comment]
I keep getting :
`---- This analysis has been done using snakePipes version 2.1.0 ----
Job counts:
	count	jobs
	1	FASTQ1
	1

---- This analysis has been done using snakePipes version 2.1.0 ----
Job counts:
	count	jobs
	1	FASTQ1
	1

---- This analysis has been done using snakePipes version 2.1.0 ----
Job counts:
	count	jobs
	1	FASTQ2
	1
`

for every single fastq job. Was there any intention or it should get fixed?

**Comment by katsikora** [comment]
From the logs I've seen up till now, this statement  is printed only once per snakemake execution and I've never seen it being appended multiple times like that. 

It looks like something's wrong with your workflow - why do you get only job count of one?
Yes, I think this should get fixed, whatever's causing that.

**Comment by LeilyR** [comment]
I send you a link to log file with that issue

**Comment by katsikora** [comment]
Sofar, we've been able to narrow the issue down to local rules FASTQ1 and FASTQ2 and to exclude snakemake version increase as cause.

**Comment by katsikora** [comment]
Moving these rules to the cluster (i.e. removing the localrules declaration) fixes the issue.

**Comment by LeilyR** [comment]
This rule also prints loads of other info. out such as genome info. each time it is called.

**Comment by katsikora** [comment]
This is not up to date anymore, I'm closing.
